src/compile/axis.ts

```diff
@@ -186,6 +186,14 @@ export function tickSize(model: Model, channel: Channel) {
   return undefined;
 }
 
+export function tickSizeEnd(model: Model, channel: Channel) {
```

Comment:
Please also move `tickSizeEnd` above from the group below
`    // b) properties without rules, only produce default values in the schema, or explicit value if specified
`

to 

`    // a) properties with special rules (so it has axis[property] methods) -- call rule functions
`

See around line 72.  (We sort each group by alphabetical order)